### PR TITLE
Fix the config error in restore

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve? 
Fix the config error in restore

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
Fix the config error in restore
```

### CODE REVIEW

File: cmd/backup-manager/app/import/restore.go [hunk 1]
```diff
@@ -58,14 +58,14 @@ func (ro *Options) loadTidbClusterData(restorePath string) error {
...
 	args := []string{
-		"--status-addr=0.0.0.0:8289",
-		"--backend=tidb",
-		"--server-mode=false",
-		"–-log-file=",
-		fmt.Sprintf("--tidb-user=%s", ro.User),
-		fmt.Sprintf("--tidb-password=%s", ro.Password),
-		fmt.Sprintf("--tidb-host=%s", ro.Host),
-		fmt.Sprintf("--d=%s", restorePath),
+		"-status-addr=0.0.0.0:8289",
```
Comment (DanielZhangQD) [inline cmd/backup-manager/app/import/restore.go:61]: The options in the tidb-lightning chart are also with `--` prefix, could you please double confirm the format? 
https://github.com/pingcap/tidb-operator/blob/master/charts/tidb-lightning/templates/scripts/_start_lightning.sh.tpl#L28

File: cmd/backup-manager/app/import/restore.go [hunk 2]
```diff
@@ -61,7 +61,8 @@ func (ro *Options) loadTidbClusterData(restorePath string) error {
 		"--status-addr=0.0.0.0:8289",
 		"--backend=tidb",
 		"--server-mode=false",
-		"–-log-file=",
+		"--log-file=",
+		"--tidb-port=4000",
```
Comment (DanielZhangQD) [inline cmd/backup-manager/app/import/restore.go:65]: Please use `ro.Port`.

Comment (Yisaer) [reply cmd/backup-manager/app/import/restore.go]: updated.

**Pull request merged**